libs/chain-signatures/contract/src/primitives/participants.rs

```diff
@@ -174,6 +186,20 @@ impl Participants {
             self.participants.remove(pos);
         }
     }
+
+    pub fn change_info(
```

Comment:
Nit(name suggestion): I'd call this method `update_info`. It feels more conventional but I don't have a motivation for why at the top of my head.

```suggestion
    pub fn update_info(
```